workitem/workitem_repository.go

```diff
@@ -17,10 +17,15 @@ import (
 	uuid "github.com/satori/go.uuid"
 )
 
+const orderValue = 1000
+const above = "above"
+const below = "below"
+
 // WorkItemRepository encapsulates storage & retrieval of work items
 type WorkItemRepository interface {
 	Load(ctx context.Context, ID string) (*app.WorkItem, error)
 	Save(ctx context.Context, wi app.WorkItem) (*app.WorkItem, error)
+	Reorder(ctx context.Context, position *app.WorkItemReorderPosition, wi app.WorkItem) (*app.WorkItem, error)
```

Comment:
@aslakknutsen To calculate the reorder item's position (https://github.com/almighty/almighty-core/pull/804/files#diff-55f2da878ed2fcb42e5373010f02ff92R210)

I am passing rest API workitem object. So, why not rest API position object?